infrastructure/grpc/server_test.go

```diff
@@ -0,0 +1,495 @@
+package grpc_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/gomega"
+	grpcPkg "google.golang.org/grpc"
+
+	mvm1 "github.com/weaveworks/flintlock/api/services/microvm/v1alpha1"
+	"github.com/weaveworks/flintlock/api/types"
+	"github.com/weaveworks/flintlock/core/models"
+	"github.com/weaveworks/flintlock/infrastructure/grpc"
+	"github.com/weaveworks/flintlock/infrastructure/mock"
+)
+
+func TestServer_CreateMicroVM(t *testing.T) {
```

Comment:
I don't think the validation call is covered here?

Reply:
It's not. I was unsure the best way to go about that - given the validation package itself is tested I didn't know if it was necessary to test the call to that package in this file?